proposals/jib-cli-surface.md

```diff
@@ -0,0 +1,60 @@
+# Jib CLI Surface Specification
+
+Specification for a command structures exposed to users of the jib-cli
+
+## Usage
+`jib COMMAND [OPTIONS]`
+
+## Commands
+
+The jib cli exposes the following top level commands, mirroring what is available in the plugins
+```
+build        build to registry
+dockerBuild  build to docker daemon
+buildTar     build to a tar file
+```
+
+## Options
+
+### Required
+```
+-t  --target <image-ref>  a target image reference
+```
+
+### Optional
+```
+-b, --build-file <file>                    location of the build file (default <context>/jib.yaml)
+-c, --context <dir>                        location of the build context (default .)
+    --credHelper <registry>=<credHelper>   credential helper to use for a registry (repeatable)
+    --docker-config <directory>            location of docker configuration
+    --help                                 print usage
+-p, --parameter <name>=<value>             templating parameters replace `${name}` with `value` in the build file (repeatable)
+    --tag <tag1>[,<tag2>,...]              additional tags for target
+    --verbosity <level>                    set logging verbosity (error, warn, lifecycle (default), info, debug)
+-v, --version                              print version information and exit 
```

Comment:
Is this the general jib-cli version?

Reply:
yeah this would be whatever version information we bake into the system.

Reply:
nice. we could add this maybe?
```suggestion
-v, --version                              print jib-cli version information and exit 
```

Reply:
I think when you run `tool --version`, it's expected that you are expecting the version of the tool, no?